recipes/cppcheck/all/test_package/conanfile.py

```diff
@@ -1,20 +1,31 @@
-from conans import ConanFile, tools
+from conan import ConanFile
+from conan.tools.build import can_run
+from conans.tools import get_env
```

Comment:
Is there a `get_env` on the `conan` namespace? I couldn't find it so I kept the old one from `conans`